experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/filter/BitmapFilter.desktop.kt

```diff
@@ -46,27 +63,23 @@ fun applyGrayScaleFilter(bitmap: BufferedImage): BufferedImage {
     return result
 }
 
-fun applyPixelFilter(bitmap: BufferedImage): BufferedImage {
-
+private fun applyPixelFilter(bitmap: BufferedImage): BufferedImage {
     val w: Int = bitmap.width
     val h: Int = bitmap.height
-
-    var result = scaleBitmapAspectRatio(bitmap, w / 4, h / 4)
+    var result = scaleBitmapAspectRatio(bitmap, w / 10, h / 10)
```

Comment:
I see value `12` on another platform, why it differs?

Reply:
It doesn't have big sense. Just to apply proper "pixelize" effect.
This value may differ, or may be the same.

Reply:
I changed to 12